app/app.go

```diff
@@ -587,14 +583,14 @@ func New(
 	// 	consensusRegistry,
 	// )
 
-	// app.EvmKeeper = *evmmodulekeeper.NewKeeper(
-	// 	appCodec,
-	// 	keys[evmmoduletypes.StoreKey],
-	// 	memKeys[evmmoduletypes.MemStoreKey],
-	// 	app.GetSubspace(evmmoduletypes.ModuleName),
-	// 	app.ConsensusKeeper,
-	// 	app.ValsetKeeper,
-	// )
+	app.EvmKeeper = *evmmodulekeeper.NewKeeper(
+		appCodec,
+		runtime.NewKVStoreService(keys[evmmoduletypes.StoreKey]),
+		memKeys[evmmoduletypes.MemStoreKey],
+		nil,
+		nil,
```

Comment:
Why these values are nil? Remove these fields if not needed or add TODO comment if they need to be updated later.